SurveyApp/Extensions/UIKit/UIFont/UIFont+Size.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  UIFont+Size.swift
+//  SurveyApp
+//
+//  Created by Chananchida F. on 6/8/21.
+//
+
+import UIKit
+
+extension UIFont {
+    
+    enum FontSize: CGFloat {
+        /**size = 46*/
```

Comment:
should use `///` for single line documentation comments `/** */` is used for multiline comments